frappe/commands/site.py

```diff
@@ -1128,6 +1142,15 @@ def start_ngrok(context, bind_tls):
 	site = get_site(context)
 	frappe.init(site=site)
 
+	ngrok_authtoken = frappe.conf.ngrok_authtoken
```

Comment:
Haven't checked the docs for ngrok yet but i'm hoping we can also prompt the user for passing the auth token (if not present in site_config or if it's present, it can be considered as default and user can still enter a different one if they want)?

https://click.palletsprojects.com/en/8.1.x/options/#password-prompts
https://click.palletsprojects.com/en/8.1.x/api/#click.prompt